cardano-db-sync/src/Cardano/DbSync/Util.hs

```diff
@@ -139,13 +140,17 @@ pointToSlotHash (Point x) =
     Origin -> Nothing
     At blk -> Just (Ledger.SlotNumber . unSlotNo $ Point.blockPointSlot blk, unByronHash $ Point.blockPointHash blk)
 
-renderAbstractHash :: ByteArrayAccess bin => bin -> Text
-renderAbstractHash =
+renderByteArrayToText :: ByteArrayAccess bin => bin -> Text
+renderByteArrayToText =
```

Comment:
There is no function `renderByteArrayToString` so 'ToText` is unnecesary.

The convention in this code base is `renderX :: X -> Type`.